ray-operator/test_log_1

```diff
@@ -0,0 +1,1275 @@
+2023-12-08T18:00:45Z	INFO	controllers.RayService	RayCluster rayservice-sample-raycluster-g79cg's headService has already exists, skip Update
+2023-12-08T18:00:45Z	INFO	controllers.RayService	reconcileServices	{"serviceType": "serveService", "RayService name": "rayservice-sample", "RayService namespace": "default"}
+2023-12-08T18:00:45Z	INFO	controllers.RayService	reconcileServices	{"newSvc": "&Service{ObjectMeta:{rayservice-sample-serve-svc  default    0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[ray.io/serve:rayservice-sample-serve ray.io/service:rayservice-sample] map[] [] [] []},Spec:ServiceSpec{Ports:[]ServicePort{ServicePort{Name:serve,Protocol:,Port:8000,TargetPort:{0 0 },NodePort:0,AppProtocol:nil,},},Selector:map[string]string{ray.io/cluster: rayservice-sample-raycluster-g79cg,ray.io/serve: true,},ClusterIP:,Type:,ExternalIPs:[],SessionAffinity:,LoadBalancerIP:,LoadBalancerSourceRanges:[],ExternalName:,ExternalTrafficPolicy:,HealthCheckNodePort:0,PublishNotReadyAddresses:false,SessionAffinityConfig:nil,IPFamilyPolicy:nil,ClusterIPs:[],IPFamilies:[],AllocateLoadBalancerNodePorts:nil,LoadBalancerClass:nil,InternalTrafficPolicy:nil,},Status:ServiceStatus{LoadBalancer:LoadBalancerStatus{Ingress:[]LoadBalancerIngress{},},Conditions:[]Condition{},},}"}
+2023-12-08T18:00:45Z	INFO	controllers.RayService	RayCluster rayservice-sample-raycluster-g79cg's serveService has already exists, skip Update
+2023-12-08T18:00:45Z	DEBUG	events	The Serve applicaton is now running and healthy.	{"type": "Normal", "object": {"kind":"RayService","namespace":"default","name":"rayservice-sample","uid":"4a8484cc-467c-4f14-8a8c-1aacfcfebd2a","apiVersion":"ray.io/v1","resourceVersion":"263"}, "reason": "Running"}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	reconciling RayJob	{"NamespacedName": {"name":"rayjob-test-default-2","namespace":"default"}}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	initRayJobStatusIfNeed	{"shouldUpdateStatus": false, "RayJob": "rayjob-test-default-2", "jobId": "rayjob-test-default-2-b4g78", "rayClusterName": "rayjob-test-default-raycluster-kvjgm", "jobStatus": "PENDING"}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	Found associated RayCluster for RayJob	{"rayjob": "rayjob-test-default-2", "raycluster": {"name":"rayjob-test-default-raycluster-kvjgm","namespace":"default"}}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	ClusterSelector is being used to select an existing RayCluster. RayClusterSpec will be disregarded	{"raycluster": {"name":"rayjob-test-default-raycluster-kvjgm","namespace":"default"}}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	Ray dashboard is ready	{"DashboardURL": "rayjob-test-default-raycluster-kvjgm-head-svc.default.svc.cluster.local:8265"}
+2023-12-08T18:00:45Z	INFO	controllers.RayJob	waiting for the cluster to be ready	{"rayCluster": "rayjob-test-default-raycluster-kvjgm"}
+[38;5;10m•[0m[38;5;10m•[0mSetup mock first: 2023-12-08 18:00:45.476819512 +0000 UTC m=+18.245897848
```

Comment:
`utils.GetRayDashboardClientFunc().(*utils.FakeRayDashboardClient).GetJobInfoMock.Store(&mockedGetJobInfo)` is called.